docs/reference/ml/ml-shared.asciidoc

```diff
@@ -507,6 +507,355 @@ tag::data-frame-analytics-stats[]
+heuristics.
++
+.Properties of `parameters`
+[%collapsible%open]
+======
+`compute_feature_influence`::::
+(boolean)
+If true, feature influence calculation is enabled.
+
+`feature_influence_threshold`::::
+(double)
+The minimum {olscore} that a document needs to have to calculate its feature 
+influence score.
+
+`method`::::
+(string)
+The method that {oldetection} uses. Possible values are `lof`, `ldof`, 
+`distance_kth_nn`, `distance_knn`, and `ensemble`.
+
+`n_neighbors`::::
+(integer)
+The value for how many nearest neighbors each method of {oldetection} uses to 
+calculate its outlier score.
+
+`outlier_fraction`::::
+(double)
+The proportion of the data set that is assumed to be outlying prior to 
+{oldetection}.
+
+`standardization_enabled`::::
+(boolean)
+If true, then the following operation is performed on the columns before 
+computing {olscores}: (x_i - mean(x_i)) / sd(x_i).
+======
+//End parameters
+
+`timestamp`::::
+(date)
+The timestamp when the statistics returned in milliseconds since the epoch.
+
+//Begin od_timing_stats
+`timing_stats`::::
+(object)
+An object containing time statistics about the {dfanalytics-job}.
++
+.Property of `timing_stats`
+[%collapsible%open]
+======
+`elapsed_time`::::
+(integer)
+Runtime of the {dfanalytics-job} in milliseconds.
+======
+//End od_timing_stats
+=====
+//End outlier_detection_stats
+
+//Begin regression_stats
+`regression_stats`:::
+(object)
+An object containing statistical data about the {reganalysis}.
++
+.Properties of `regression_stats`
+[%collapsible%open]
+=====
+//Begin reg_hyperparameters
+`hyperparameters`::::
+(object)
+An object containing the paramters of the {reganalysis}.
++
+.Properties of `hyperparameters`
+[%collapsible%open]
+======
+`alpha`::::
+(double)
+Regularization factor to penalize deeper trees when training decision trees.
+
+`downsample_factor`::::
+(double)
+The value of the downsample factor.
+
+`eta`::::
+(double)
+The value of the eta hyperparameter.
+
+`eta_growth_rate_per_tree`::::
+(double)
+Specifies with what rate to increase `eta` for every new tree added to the 
+forest. The rate of 1.0 does not change `eta`, the rate of 1.05 increases `eta` 
+by 5%.
+
+`feature_bag_fraction`::::
+(double)
+The fraction of features that is used when selecting a random bag for each 
+candidate split.
+
+`gamma`::::
+(double)
+Regularization factor to penalize trees with large numbers of nodes.
+
+`lambda`::::
+(double)
+Regularization factor to penalize large leaf weights.
+
+`max_attempts_to_add_tree`::::
+(integer)
+If the algorithm fails to determine a non-trivial tree (more than a single 
+leaf), this parameter determines how many of such consecutive failures are 
+tolerated. Once the number of attempts exceeds the threshold, the forest 
+training stops.
+
+`max_optimization_rounds_per_hyperparameter`::::
+(integer)
+A multiplier responsible for determining the maximum number of 
```

Comment:
> A multiplier responsible ... 

This content seems to duplicate an earlier definition.  Can the content be re-used?

Reply:
@lcawl Yes, it can be. I single-sourced all the possible shared properties in a7bec16.